enterprise_access/apps/api/v1/views/utils.py

```diff
@@ -1,19 +1,59 @@
 """
 Utilities for REST API View and Viewset classes.
 """
-from rest_framework.pagination import PageNumberPagination
+from django.conf import settings
+from edx_rest_framework_extensions.paginators import DefaultPagination
 
 
-class PaginationWithPageCount(PageNumberPagination):
+class PaginationWithPageCount(DefaultPagination):
```

Comment:
By migrating from `PageNumberPagination` (DRF standard) -> `DefaultPagination` (Open edX standard), list APIs throughout enterprise-access will have additional properties such as `num_pages` and `current_page` without extra effort.

However, it does mean we have to override `DefaultPagination`'s `page_size = 10` to keep the original `page_size` for `PaginationWithPageCount` in place (i.e., relies on the `REST_FRAMEWORK.PAGE_SIZE` Django settings.